src/assets/content/dictionary/SPA.md

```diff
@@ -0,0 +1,10 @@
+---
+title: SPA
+definition: SPA stands for Single Page Application. It is a very common way of programming websites these days. The idea is that the website loads all the HTML/JS the first time you visit. When you then navigate, the browser will only rerender the content without refreshing the website.
+sources:
+- sourceurl: https://betterprogramming.pub/how-to-implement-a-single-page-application-using-react-router-cc6b9e5c3aec#:~:text=SPA%20stands%20for%20Single%20Page,content%20without%20refreshing%20the%20website.
+perspectives:
+- meaning: SPA is used to make the user experience feel a lot smoother. You can tell when it’s a SPA or multi-page application when navigating between menus often because a multi-page application will reload, making the whole UI blink fast depending on the content.
+  role: As a developer
```

Comment:
```suggestion
  role: developer
```
suggestion: Remove the "As a" so that the sentence will concat correctly. You can accept the changes here via the Github UI, or make an additional commit, as you like.